app/templates/partials/navigation.html

```diff
@@ -6,7 +7,8 @@
       {% if nav_item.current_location %}
         <div>{{nav_item.link_name}}</div>
       {% else %}
-        <a class="nav__link" href="{{nav_item.link_url}}">{{nav_item.link_name}}</a>
+        <a class="nav__link" href="{{nav_item.link_url}}"
+          {{helpers.track('click', 'Navigation', 'Section link', nav_item.link_url)}}>{{nav_item.link_name}}</a>
```

Comment:
Did you decide not to make the action's verbs in the end? e.g. `Section Link Clicked` ?